src/tsdb.c

```diff
@@ -464,6 +463,28 @@ void FreeSeries(void *value) {
     free(series);
 }
 
+int DefragSeries(RedisModuleDefragCtx *ctx, RedisModuleString *key, void **value) {
```

Comment:
Could it just be a `void` if we always return just `REDISMODULE_OK`?

Reply:
The signature is dictated by Redis. This is a callback.

Reply:
Ah, I apologise then, I missed that it was a callback; thought it was our function.